app/src/main/java/com/amaze/filemanager/utils/Utils.java

```diff
@@ -241,8 +262,8 @@ public static int nthToLastCharIndex(int elementNumber, String str, char element
      * Formats input to plain mm:ss format
      */
     public static String formatTimer(long timer) {
-        final long min = TimeUnit.SECONDS.toMinutes(timer);
-        final long sec = TimeUnit.SECONDS.toSeconds(timer - TimeUnit.MINUTES.toMillis(min));
-        return String.format("%02d:%02d", min, sec);
+        final long min = timer / 60;
+        final long sec = timer % 60;
+        return String.format(Locale.US, "%02d:%02d", min, sec);
```

Comment:
Why are you hardcoding a locale? doesn't the Android System manage locales acordingly?

Reply:
Per recommendation from Android Studio's lint. As long as this is simply `mm:ss`, it should be fine.

Unless this will change to locale-specific strings (e.g. mm分ss秒 in Chinese)...

Reply:
> Per recommendation from Android Studio's lint. As long as this is simply `mm:ss`, it should be fine.
>
> Unless this will change to locale-specific strings (e.g. mm分ss秒 in Chinese)...

It should be fine, but I don't know what transformations Java does based on the locale, and would not like to introduce subtle bugs, irreplicable bugs (see section "Be wary of the default locale" [here](https://developer.android.com/reference/java/util/Locale)).

I see two solutions, using the `SimpleDateTime` class and using [this fix](https://stackoverflow.com/questions/14389349/android-get-current-locale-not-default) from StackOverflow. I would also accept that you return the code to String.format("%02d:%02d", min, sec) so that lint keeps on warning.